src/datasources/results/components/query-builders/query-results/ResultsQueryBuilder.test.tsx

```diff
@@ -55,6 +55,15 @@ describe('ResultsQueryBuilder', () => {
       expect(conditionsContainer.item(0)?.textContent).toContain("PASSED"); //value
     });
 
+    it('should select keyword in query builder', () => {
+      const { conditionsContainer } = renderElement([], [], [], 'Keywords.Contains("keyword1")');
+
+      expect(conditionsContainer?.length).toBe(1);
+      expect(conditionsContainer.item(0)?.textContent).toContain("Keyword"); //label
```

Comment:
```suggestion
      expect(conditionsContainer.item(0)?.textContent).toContain("Keywords"); //label
```
Why is the name in the property dropdown `Keyword` and not `Keywords`?

Reply:
In SLE Results query builder, the label is `Keyword`
![image](https://github.com/user-attachments/assets/5958ce41-ffab-483a-bbfb-2813d3b6379f)

But in results query builder of dataspace query it is `Keywords`. Should I provide labels similar to dataspace or results query builder?

Reply:
I am okay with keeping it as Keyword